src/dom/Window.js

```diff
@@ -64,15 +64,15 @@ export class Window extends EventTarget {
 
 let lastTime = 0
 const requestAnimationFrame = callback => {
-  const now = new global.Date().getTime()
+  const now = new globalThis.Date().getTime()
```

Comment:
Same question for globalThis :D

Reply:
Looks like globalThis has been around since Node 12: https://blog.logrocket.com/what-is-globalthis-why-use-it/

Here's the info on protocol specifiers: https://2ality.com/2021/12/node-protocol-imports.html